lib/mailchimp_helper.rb

```diff
@@ -9,9 +9,13 @@ def mailchimp_snippet
     def set_snippet
       return unless mailchimp_store_id && ::SpreeMailchimpEcommerce.configuration.mailchimp_api_key
 
-      Rails.cache.fetch "mailchimp_settings_#{mailchimp_store_id}" do
-        ::Gibbon::Request.new(api_key: ::SpreeMailchimpEcommerce.configuration.mailchimp_api_key).
-          ecommerce.stores(mailchimp_store_id).retrieve.body["connected_site"]["site_script"]["fragment"]
+      begin
+        Rails.cache.fetch "mailchimp_settings_#{mailchimp_store_id}" do
+          ::Gibbon::Request.new(api_key: ::SpreeMailchimpEcommerce.configuration.mailchimp_api_key).
+            ecommerce.stores(mailchimp_store_id).retrieve.body["connected_site"]["site_script"]["fragment"]
+        end
+      rescue Gibbon::MailChimpError => e
+        ""
```

Comment:
@chhga could we add some logger here? So we're aware that something bad happened? thanks